keyboards/enviousdesign/60f/info.json

```diff
@@ -0,0 +1,227 @@
+                {"label": "1,11", "matrix": [1, 11], "x": 13, "y": 1},
+                {"label": "1,12", "matrix": [1, 12], "x": 14, "y": 1},
+                {"label": "1,13", "matrix": [1, 13], "x": 15, "y": 1},
+                {"label": "2,0", "matrix": [2, 0], "x": 2.5, "y": 2, "w": 1.75},
+                {"label": "2,2", "matrix": [2, 2], "x": 4.25, "y": 2},
+                {"label": "2,3", "matrix": [2, 3], "x": 5.25, "y": 2},
+                {"label": "2,4", "matrix": [2, 4], "x": 6.25, "y": 2},
+                {"label": "2,5", "matrix": [2, 5], "x": 7.25, "y": 2},
+                {"label": "2,6", "matrix": [2, 6], "x": 8.25, "y": 2},
+                {"label": "2,7", "matrix": [2, 7], "x": 9.25, "y": 2},
+                {"label": "2,8", "matrix": [2, 8], "x": 10.25, "y": 2},
+                {"label": "2,9", "matrix": [2, 9], "x": 11.25, "y": 2},
+                {"label": "2,10", "matrix": [2, 10], "x": 12.25, "y": 2},
+                {"label": "2,11", "matrix": [2, 11], "x": 13.25, "y": 2},
+                {"label": "2,12", "matrix": [2, 12], "x": 14.25, "y": 2},
+                {"label": "2,13", "matrix": [2, 13], "x": 15.25, "y": 2},
+                {"label": "Enter", "matrix": [2, 14], "x": 16.25, "y": 1, "w": 1.25, "h": 2},
+                {"label": "3,0", "matrix": [3, 0], "x": 2.5, "y": 3, "w": 1.25},
+                {"label": "3,1", "matrix": [3, 1], "x": 3.75, "y": 3},
+                {"label": "3,2", "matrix": [3, 2], "x": 4.75, "y": 3},
+                {"label": "3,3", "matrix": [3, 3], "x": 5.75, "y": 3},
+                {"label": "3,4", "matrix": [3, 4], "x": 6.75, "y": 3},
+                {"label": "3,5", "matrix": [3, 5], "x": 7.75, "y": 3},
+                {"label": "3,6", "matrix": [3, 6], "x": 8.75, "y": 3},
+                {"label": "3,7", "matrix": [3, 7], "x": 9.75, "y": 3},
+                {"label": "3,8", "matrix": [3, 8], "x": 10.75, "y": 3},
+                {"label": "3,9", "matrix": [3, 9], "x": 11.75, "y": 3},
+                {"label": "3,10", "matrix": [3, 10], "x": 12.75, "y": 3},
+                {"label": "3,11", "matrix": [3, 11], "x": 13.75, "y": 3},
+                {"label": "3,13", "matrix": [3, 13], "x": 14.75, "y": 3, "w": 2.75},
+                {"label": "4,0", "matrix": [4, 0], "x": 2.5, "y": 4, "w": 1.25},
+                {"label": "4,1", "matrix": [4, 1], "x": 3.75, "y": 4, "w": 1.25},
+                {"label": "4,3", "matrix": [4, 3], "x": 5, "y": 4, "w": 1.25},
+                {"label": "4,6", "matrix": [4, 6], "x": 6.25, "y": 4, "w": 6.25},
+                {"label": "4,10", "matrix": [4, 10], "x": 12.5, "y": 4, "w": 1.25},
+                {"label": "4,11", "matrix": [4, 11], "x": 13.75, "y": 4, "w": 1.25},
+                {"label": "4,13", "matrix": [4, 13], "x": 15, "y": 4, "w": 1.25},
+                {"label": "4,14", "matrix": [4, 14], "x": 16.25, "y": 4, "w": 1.25}
+            ]
+        },
+        "LAYOUT_all": {
```

Comment:
based on PCB image, `LAYOUT_all` is missing split Backspace and split Right Shift

Reply:
The updated version with the split keys have yet to be ordered.

Reply:
@Envious-Data,

Are you intending for the updated version to have its own revision?

Reply:
not sure yet, If its possible to make it backwards compatible there wont be a revision folder added